### ISSUE TITLE
LoadError with compiled multi-byte symbol

### ISSUE THREAD

**Issue by jakago** [jruby/jruby#4659]
multibytesymbol.rb
```
# coding: utf-8

# :α.to_s.bytes => [206, 177]
p :α  
```
run with jruby 9.1.9.0
```
$ jruby multibytesymbol.rb
:α
```
compile and run with jruby 9.1.9.0
```
$ jrubyc multibytesymbol.rb
$ jruby multibytesymbol.class
LoadError: multibytesymbol.class is not compiled Ruby; use java_import to load normal classes
```

**Comment by headius** [comment]
Again another possible persistence bug. @enebo

**Comment by enebo** [comment]
The fix from #4662 sort of makes this no longer totally fail but it is broken because we save a string without knowing the encoding and then read it back into the file.coding j.l.String.  Symbol assumes iso8859_1 so it gets bungled.

I marked this against 9.2 since we have 2 side branches changing our internals to be based on either (ByteList or RubySymbol - an ongoing discussion).  What we have to date is pushing a ByteList through the parser.   So we have proper encoding info at our disposal we just need to convert the planet to using what format we pick and all will be fixed for good.

**Comment by headius** [comment]
Fixed in 9.2 by @enebo symbol work.
